Xamarin.Forms.Core/MediaElementState.cs

```diff
@@ -0,0 +1,12 @@
+namespace Xamarin.Forms
+{
+	public enum MediaElementState
+	{
+		Closed = 0,
+		Opening = 1,
+		Buffering = 2,
+		Playing = 3,
+		Paused = 4,
+		Stopped = 5,
+	}
```

Comment:
Is it necessary to assign default numbers in this enum?

Reply:
In an older version these were to match with the UWP values. It's no longer used like that so I'll remove them.